helm/alfresco-process-infrastructure/setup_acs.sh

```diff
@@ -7,10 +7,10 @@ add_user_to_group "service-account-storage-service" "ALFRESCO_ADMINISTRATORS"
 
 if [[ "$LOAD_TEST_DATA" == "true" ]]
 then
-  create_user "admin.adf" "admin.adf@alfresco.com"
+	create_user "admin.adf" "admin.adf@alfresco.com" $(generate_alfpwd)
```

Comment:
we should do for the service-account user for PaaS, test data is for testing and can stay with default